3rdparty/mbedtls/ssl_client.c

```diff
@@ -68,7 +67,7 @@ static void my_debug( void *ctx, int level,
     fflush(  (FILE *) ctx  );
 }
 
-int edgeless_ttls_test_client(const char* cli_crt, const char* cas_pem, const char* cli_key, bool client_auth )
```

Comment:
Is there any more documentation needed (e.g. with a "//EDG: " comment) or is the modification of the function header enough?